sdk/test-utils/recorder/src/recorder.ts

```diff
@@ -91,6 +92,32 @@ export function record(testContext: Mocha.Context): Recorder {
         recorder.stop();
       }
     },
+    /**
+     * `{recorder.skip("node")}` and `{recorder.skip("browser")}` will skip the test in node.js and browser runtimes repectively.
+     * If the `{runtime}` is undefined, the test will be skipped in both the node and browser runtimes.
+     * @param runtime Can either be "node" or "browser"
+     */
+    skip: function(runtime?: "node" | "browser"): void {
+      // 1. skipping the test only in node
+      // 2. skipping the test only in browser
+      // 3. skipping the test in both the node and browser runtimes
```

Comment:
The type you're using doesn't allow for #3.  You'd need a `| "both"`

Reply:
How do you say to skip the test in both, just not pass nothing?  I guess that's okay, yeah.